src/java/exception/MaxPassengerExceededException.java

```diff
@@ -9,6 +9,7 @@
     public MaxPassengerExceededException(int maxPassengers, int expectedPassengersLoad) {
         this.maxPassengers = maxPassengers;
         this.expectedPassengersLoad = expectedPassengersLoad;
+        System.out.println("Za dużo pasażerów! Maksymalna ilość miejsc wynosi "+ getMaxPassengers());
```

Comment:
It won't cause any problems. Check [documentation](https://docs.oracle.com/javase/8/docs/api/java/lang/Exception.html) of `Exception` class. 

> The class Exception and its subclasses are a form of Throwable that indicates **conditions that a reasonable application might want to catch.** 

There is nothing about handling, only **conditions to catch**.